ext/sodium/LICENSE

```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2013-2017, Frank Denis
```

Comment:
Shouldn't all core extension be under the PHP license? I know that there are some files under other licenses but is there actually a core extension with other primary license than the PHP license. If this would be the first one I think that this would be good to mention on the internal list unless it's been already discussed and I just missed it...